### PR TITLE
added auto-documentation for errorCodes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lib/views/docs.jade [hunk 1]
```diff
@@ -56,7 +56,10 @@ mixin props(properties)
                     dl.meta
                         each key in meta
                             dt= key
-                            dd= details[key]
+                            if key === 'errorCodes'
+                                dd= JSON.stringify(details[key], null, 2).substr(3, JSON.stringify(details[key], null, 2).length -5)
```
Comment (djvirgen) [inline lib/views/docs.jade]: We're stringifying twice. How about iterating over the values and creating a new `dd` for each?

Comment (ghost) [inline lib/views/docs.jade]: modified to slice so we eliminate stringifying twice :)

**Comment by davidkaminsky** [comment]
Any chance you can post a visual of what this'll look like now?

**Comment by ghost** [comment]
The code only affects the `errorCodes` property and results in something similar to:

```
errorCodes    {
                "code": 1000,
                "error": "INVALID",
                "message": "Country code is not valid"
              }
```

Previously, the code read: errorCodes    `[object Object]`

**Comment by djvirgen** [comment]
LGTM :cherries:

**Comment by ghost** [comment]
yay!

**Pull request merged**